tensorflow_addons/version.py

```diff
@@ -24,7 +24,7 @@
 # stable release (indicated by `_VERSION_SUFFIX = ''`). Outside the context of a
 # release branch, the current version is by default assumed to be a
 # 'development' version, labeled 'dev'.
-_VERSION_SUFFIX = "dev"
+_VERSION_SUFFIX = ""
 
 # Example, '0.1.0-dev'
```

Comment:
Could we change also the example?

Reply:
Could you elaborate on what you want it to be change to?

Reply:
Too late It Is merged :smile:
Nothing important It Is just to remove the -dev suffix in the example on that line.